StardewArchipelago/Archipelago/ArchipelagoClient.cs

```diff
@@ -78,6 +78,12 @@ public void Connect(ArchipelagoConnectionInfo connectionInfo, out string errorMe
                 return;
             }
 #endif
+
+            if (!SlotData.Mods.IsPatcherStateCorrect(_modHelper, out errorMessage))

```

Comment:
can you put that in the `#if RELEASE` block, so that we don't have to deal with it when debugging, like the mod state?